tt_metal/common/blockfloat_common.hpp

```diff
@@ -53,6 +53,25 @@ inline uint32_t get_exp_dword(const std::vector<uint8_t>& vec) {
     return tmp;
 }
 
+inline std::vector<uint32_t> pack_exponents(const std::vector<uint8_t>& exponents, size_t num_elements_in_dword) {
+    TT_FATAL(
+        exponents.size() % num_elements_in_dword == 0, "Input vector size {} must be divisible by 4", exponents.size());
```

Comment:
Did you mean to say by 4 or something else, like num_elements_in_dword (until and unless it is always 4)

Reply:
done (always 4 though).

I'll add the intent of my code here:
so we need to make the exp section L1 aligned, this means when creating the vector we need to insert paddings to the data vector.